environment-pipelines/iam.tf

```diff
@@ -606,7 +607,7 @@ data "aws_iam_policy_document" "postgres" {
 }
 
 resource "aws_iam_policy" "postgres" {
-  name        = "postgres-access"
+  name        = "${var.application}-${var.pipeline_name}-postgres-access"
```

Comment:
Should we include the word "pipeline" in these to avoid risk of future clashes where a pipeline shares the same name as an environment?

Reply:
What sort of clashes are you thinking about? I might be mistaken in my understanding, but what difference does it make in adding the `pipeline` word other than being verbose?

Example:
`dev-dev-postgres-access` and `dev-piepline-dev-postgres-access` -> not sure how the word `pipeline` helps with name clash? For readability purposes, I agree yes